src/Rector/ClassMethod/CommandConstantReturnCodeRector.php

```diff
@@ -118,7 +118,7 @@ private function convertNumberToConstant(LNumber $lNumber): ?ClassConstFetch
         }
 
         return $this->nodeFactory->createShortClassConstFetch(
```

Comment:
Oops, it should be `createClassConstFetch` instead of `createShortClassConstFetch`, I will make a patch :)

Reply:
https://github.com/rectorphp/rector-symfony/pull/297